vendor/bat-native-ads/include/bat/ads/ads_client.h

```diff
@@ -60,6 +60,29 @@ class ADS_EXPORT AdsClient {
   // Should return the maximum number of ads that can be shown per day
   virtual uint64_t GetAdsPerDay() const = 0;
 
+  // Should return the country subdivision
+  virtual std::string GetCountrySubdivision() const = 0;
+
+  // Should set the country subdivision
+  virtual void SetCountrySubdivision(
+      const std::string&) = 0;
```

Comment:
for consistent could we please name the argument in the interface

Reply:
https://github.com/brave/brave-core/pull/5602/files#diff-2603c096c1040e59d15556ab0596bed1R68